### ISSUE TITLE
RDS Public accessibility should be visible with the 'public' command

### ISSUE THREAD

**Issue by jleopold28** [duo-labs/cloudmapper#561]
I have been running `python cloudmapper.py public --account <acct>` on an AWS account with a publicly accessible RDS instance. 

The cloudmapper report generated via the `collect` command does show that 'RDS has a public IP address'. However, when I run the `public` command I am not getting this RDS instance in the output. 

I figured that this is because the `get_public_nodes` function is looking specifically for edges which connect to the public Internet. My RDS case is a bit specialized, as it has a SG attached which allows all traffic from instances within a separate SG.

My suggestion would be to add functionality into `get_public_nodes` which searches for rds instances that have `.node_data.PubliclyAccessible==true` and include that information in the `public` command output.

Thanks!

**Comment by 0xdabbad00** [comment]
The `public` command only shows network resources that are publicly accessible to `0.0.0.0/0`.  If your RDS is only allowing access from a specific IP range it will not show up in the results of the `public` command.
